code/modules/projectiles/projectile/bullets.dm

```diff
@@ -221,14 +221,14 @@
 					if(blocked < 100 && !(blocked < 20))
 						L.emote("yawns")
 					if(blocked < 20)
-						addtimer(CALLBACK(src, .proc/apply_sedative, target, 10), 120)
+						if(L.reagents)	L.reagents.add_reagent("stoxin", 10)

```

Comment:
I don't think stoxin is fast acting enough, wouldn't CH be better?

Reply:
During my tests it did not take more than some seconds to disable someone with those numbers.